pkg/envoytest/harness.go

```diff
@@ -68,7 +68,9 @@ func SetupEnvoy(t *testing.T, adsAddress string, portmaps ...string) {
 		for _, pm := range portmaps {
 			args = append(args, "-p", pm)
 		}
-		args = append(args, "--rm", "--entrypoint", "envoy", "docker.io/datawire/aes:1.6.2", "--config-yaml", yaml)
+		// TODO: this should be marked as an end-to-end test after we have built an image
```

Comment:
Not sure how to handle this one because we have hardcoded it, on one hand the test was always testing against a really old version...

Reply:
ffs

Reply:
https://github.com/emissary-ingress/emissary/pull/4291